sdk/core/azure-core/src/main/java/com/azure/core/util/CoreUtils.java

```diff
@@ -43,6 +45,13 @@ public final class CoreUtils {
 
     private static final char[] LOWERCASE_HEX_CHARACTERS = "0123456789abcdef".toCharArray();
 
+    private static boolean shutdownHookAccessHelper;
+
+    static {
+        shutdownHookAccessHelper = Boolean.parseBoolean(Configuration.getGlobalConfiguration()
+            .get("AZURE_CORE_UTILS_SHUTDOWN_HOOK_ACCESS_HELPER"));
```

Comment:
Should we call this config `AZURE_ENABLE_SHUTDOWN_HOOK_WITH_PRIVILEGE`? The name ending with `HELPER` returning a boolean is a bit odd.

Reply:
Sounds better to me